lookout/style/format/feature_utils.py

```diff
@@ -144,12 +144,12 @@ def to_comment(self, correct_y: int) -> Comment:
 
 
 CLS_SPACE = "<space>"
+CLS_TAB = "<tab>"
+CLS_NEWLINE = "<newline>"
```

Comment:
Why do we need to reorder these? cc @m09

Reply:
I think @warenlg want to reorder them so that they match the order in CLASSES (just for ease of reading/understanding).